bn-models/src/models/_lookups/gear-type.ts

```diff
@@ -1,4 +1,38 @@
-// TODO interface definition
+import { BaseLookup } from "../_base";
 
 // from lookups
-export type GearType = any;
+export const GearType = 'gear-type';
+
+export enum GearTypes {
+    GroundfishTrawlFootropeLessThan8 = "1",
+    GroundfishTrawlGreaterThan8 = "2",
+    MidwaterTrawl = "3",
+    DanishScottishSeine = "4",
+    OtherTrawlGear = "5",
+    Longline = "6",
+    VerticalHookAndLineGear = "7",
+    Pole = "8",
+    OtherHookAndLineGear = "9",
+    FishPot = "10",
+    PrawnTrawl = "11",
+    ShrimpTrawlSingleRigged = "12",
+    SprimpTrawlDoubleRigged = "13",
+    AllNetGearExceptTrawl = "14",
+    AllTrollGear = "15",
+    AllOtherMiscGear = "16",
+    OregeonSetBackFlatfishNet = "17",
+    PrawnTrap = "18",
+    HookAndLine = "19",
+    LonglineSnap = "20"
+}
+
+export interface GearType extends BaseLookup {
+    legacy?: {
+        lookupVal?: string;
+        programId?: number;
+        lookupId?: number;
+        lookupType?: string;
+    };
+    isEM?: boolean;
```

Comment:
I thought I had added these to BaseLookup, but I guess not.
Anyhow, I'd propose they should be in BaseLookup.

Reply:
moved to base